lib/status/status.go

```diff
@@ -208,6 +214,8 @@ type Cluster struct {
 	Endpoints Endpoints `json:"endpoints"`
 	// Extension is a cluster status extension
 	Extension `json:",inline,omitempty"`
+	// Version is the server version information.
+	Version *modules.Version `json:"version"`
```

Comment:
One more thing that occurred to me is that this will omit the client binary version in the status JSON blob captured as part of the debug report - I'd add it as well (`ClientVersion`?)